cmd/ltctl/main.go

```diff
@@ -53,21 +53,21 @@ func getConfig(cmd *cobra.Command) (*deployment.Config, error) {
 
 func main() {
 	rootCmd := &cobra.Command{
-		Use:          "lt-deployer",
+		Use:          "ltctl",
 		SilenceUsage: true,
-		Short:        "Create and destroy load test environments",
+		Short:        "Control load-test deployments",
 	}
 	rootCmd.PersistentFlags().StringP("config", "c", "", "path to the configuration file to use")
 
 	commands := []*cobra.Command{
 		{
-			Use:   "create",
-			Short: "Deploy a load test environment",
+			Use:   "deploy",
```

Comment:
Is there a reason we changed this ? We also need to update the other docs.

Reply:
I proposed the change. `create` is more specific to terraform but  `deploy` sounds more explanatory if we consider that we are doing beyond than creating.

Reply:
Yes, several docs will need updating after this. The reason for the change would be to make it more explicit what we are doing since `ltctl` might seem a bit generic and will become more so when we add `start` and `stop` functionalities. Maybe in the future we could have a subcommand.

Reply:
`ltctl create` sounds okay to me.

Reply:
I'm 50/50 on that. If you prefer `create`. it is OK.

Reply:
Clarified offline. Thanks guys.